view/theme/frio/templates/notifications/suggestions.tpl

```diff
@@ -0,0 +1,2 @@
+
+{{include file="notifications/intros.tpl"}}
```

Comment:
```suggestion
{{include file="notifications/notification.tpl"}}
```

Reply:
I think this change would be wrong, wouldn't it?

Reply:
Huh, yeah, good point, copy and paste issue.